svc/FetchService.js

```diff
@@ -232,12 +232,28 @@ export class FetchService extends HoistService {
 
         if (!ret.ok) {
             ret.responseText = await this.safeResponseTextAsync(ret);
-            throw Exception.fetchError(opts, ret);
+            const e = Exception.fetchError(opts, ret);
+            if (!XH.isSSO && isRelativeUrl && e.httpStatus === 401) {
```

Comment:
I see you're restricting this to a pretty narrow use case.  The client app I worked on that has an auth requirement will still need to extend fetchService.

But what specifically is the app/use case we're solving for here?  maybe you can tell me over chat?

Reply:
yes, agreed.  Use case discussed above -- Non-SSO mobile app that has lost its session and needs to immediately proceed to re-initialization of app.